apps/lockfile-explorer-web/src/containers/LockfileViewer/index.tsx

```diff
@@ -166,61 +167,55 @@ export const LockfileViewer = (): JSX.Element | ReactNull => {
 
   return (
     <div className={styles.ViewWrapper}>
-      <FilterBar
-        options={[
-          {
-            text: 'Projects',
-            active: activeFilters[LockfileEntryFilter.Project],
-            onClick: togglePackageView(LockfileEntryFilter.Project)
-          },
-          {
-            text: 'Packages',
-            active: activeFilters[LockfileEntryFilter.Package],
-            onClick: togglePackageView(LockfileEntryFilter.Package)
-          }
-        ]}
-      />
-      <div className={`${styles.ViewContents} ${appStyles.ContainerCard}`}>
-        <div className={styles.LockfileFilterBar}>
-          <input
-            type="search"
-            placeholder="Filter..."
-            value={activeFilters[LockfileEntryFilter.Project] ? projectFilter : packageFilter}
-            onChange={updateFilter(
-              activeFilters[LockfileEntryFilter.Project]
-                ? LockfileEntryFilter.Project
-                : LockfileEntryFilter.Package
-            )}
-          />
-        </div>
-        <div className={styles.lockfileEntriesWrapper}>
+      <div className={`${styles.ViewContents}`}>
+        <Tabs
+          items={[
+            {
+              header: 'Projects'
+            },
+            {
+              header: 'Packages'
+            }
+          ]}
+          value={activeFilters[LockfileEntryFilter.Project] ? 'Projects' : 'Packages'}
+          onChange={togglePackageView}
+        />
+        <Input
+          type="search"
+          placeholder="Filter..."
+          value={activeFilters[LockfileEntryFilter.Project] ? projectFilter : packageFilter}
+          onChange={updateFilter(
+            activeFilters[LockfileEntryFilter.Project]
+              ? LockfileEntryFilter.Project
+              : LockfileEntryFilter.Package
+          )}
+        />
+        <ScrollArea>
           {getEntriesToShow().map((lockfileEntryGroup) => (
             <LockfileEntryLi group={lockfileEntryGroup} key={lockfileEntryGroup.entryName} />
           ))}
-        </div>
+        </ScrollArea>
         {activeFilters[LockfileEntryFilter.Package] ? (
           <div className={styles.filterSection}>
-            <h5>Filters</h5>
-            <div
-              className={styles.filterOption}
-              onClick={changeFilter(
+            <Text type="h5" bold>
+              Filters
+            </Text>
+            <Checkbox
+              label="Must have side-by-side versions"
```

Comment:
The sizing of the checkboxes is slightly odd if the label gets word-wrapped:
![image](https://user-images.githubusercontent.com/4673363/213330934-ae5598d9-20ee-42af-adb1-65b379f4fbed.png)

Maybe we can set a minimum width?

Reply:
Fixed by adding min width